ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/PainlessScriptTool.java

```diff
@@ -30,8 +30,9 @@ public class PainlessScriptTool implements Tool {
 
     @Setter @Getter
     private String alias;
+    private static String DEFAULT_DESCRIPTION = "User this tool to get index information.";
```

Comment:
nit: "Use this tool to run a Painless script" (or something. does the agent learn about the tools from the descriptions?)

Reply:
yes